template/Data-Science/Prediction/Diabetes-Prediction/app.py

```
@@ -0,0 +1,54 @@
+import streamlit as st
+import pickle
+import pandas as pd
+
+st.set_page_config(
+    page_title = "Diabetes Prediction App",
+)
+
+file = open('model_objects.pkl', 'rb')
```

Ruff

Use a context manager for opening files

(SIM115)